runtime/vm/exceptiondescribe.c

```diff
@@ -309,6 +347,26 @@ findJ9ClassForROMClass(J9VMThread *vmThread, J9ROMClass *romClass, J9ClassLoader
 		}
 	} else {
 		ret = peekClassHashTable(vmThread, *resultClassLoader, J9UTF8_DATA(utfClassName), J9UTF8_LENGTH(utfClassName));
+		goto done;
+	}
+cacheresult:
```

Comment:
We need to cache the result only if the romClass is in SCC. So the code of `cashresult:` should be move insdie `if (j9shr_Query_IsAddressInCache(vm, romClass, romClass->romSize)) { } `